File: android/src/com/rartworks/engine/android/GdxAppTokens.java
package com.rartworks.engine.android;

import java.util.Map;

public class GdxAppTokens {
	// Ads
	public String adsId;

	// Google Play
	public String playStoreLink;
	public Map<Integer, String> achievementsByScore;

	// In App Billing
	public String billingKey;
	public String removeAdsSku;

	// --------
	// Setters:
	// --------

	public GdxAppTokens setAdsId(String value) {
		this.adsId = value;
		return this;
	}

	public GdxAppTokens setPlayStoreLink(String value) {
		this.playStoreLink = value;
		return this;
	}

	public GdxAppTokens setAchievementsByScore(Map<Integer, String> value) {
		this.achievementsByScore = value;
		return this;
	}

	public GdxAppTokens setBillingKey(String value) {
		this.billingKey = value;
		return this;
	}

	public GdxAppTokens setRemoveAdsSku(String value) {
		this.removeAdsSku = value;
		return this;
	}
}

File: android/src/com/rartworks/engine/android/GdxAppTokens.kt
package com.rartworks.engine.android

class GdxAppTokens {
	// Ads
	lateinit var adsId: String

	// Google Play
	lateinit var playStoreLink: String
	lateinit var achievementsByScore: Map<Int, String>

	// In App Billing
	lateinit var billingKey: String
	lateinit var removeAdsSku: String

	// --------
	// Setters:
	// --------

	fun setAdsId(value: String): GdxAppTokens {
		this.adsId = value
		return this
	}

	fun setPlayStoreLink(value: String): GdxAppTokens {
		this.playStoreLink = value
		return this
	}

	fun setAchievementsByScore(value: Map<Int, String>): GdxAppTokens {
		this.achievementsByScore = value
		return this
	}

	fun setBillingKey(value: String): GdxAppTokens {
		this.billingKey = value
		return this
	}

	fun setRemoveAdsSku(value: String): GdxAppTokens {
		this.removeAdsSku = value
		return this
	}
}
